Makefile

```diff
@@ -26,15 +26,17 @@ pkg/tests/testdata/traces-dataset.sz:
 
 .PHONY: e2e
 e2e: CURRENT_BRANCH?=$(shell git branch --show-current | sed 's#/#-#')
-e2e: DOCKER_IMAGE?=$(shell ./scripts/fallback-docker.sh $(LOCAL_DOCKER_BASE):head-ts2-pg14 $(GHCR_DOCKER_BASE):$(CURRENT_BRANCH)-ts2-pg14 $(GHCR_DOCKER_BASE):master-ts2-pg14)
+e2e: EXTENSION_VERSION=$(shell cat EXTENSION_VERSION)
```

Comment:
This feels fragile. One space after version and it will break the whole thing....